src/bundles/SellerRegistration/ApplicantSignup.json

```diff
@@ -35,8 +35,8 @@
       }
     },
     "services": {
-        "Change & transformation": true,
-        "Cyber security": true
+        "Change & transformation": false,
```

Comment:
I assumed that is dummy data and not relevant to anything else. I had to set them to false to check navigation from products when there are no services.